setup.py

```diff
@@ -16,9 +16,9 @@
         long_description = f.read()
 
     install_requirements = [
-        "gbd_mapping==3.0.6",
-        "vivarium==0.10.19",
-        "vivarium_public_health==0.10.21",
+        "gbd_mapping>=3.0.6, <4.0.0",
+        "vivarium==1.0.2",
```

Comment:
I'm about to release 1.0.3 fwiw.

Reply:
wait to merge this and update after vivarium is released.